tests/optim/utils/test_model_utils.py

```diff
@@ -9,6 +9,20 @@
 from tests.helpers.basic import BaseTest, assertTensorAlmostEqual
 
 
+class TestPadReflectiveA4D(BaseTest):
+    def test_pad_reflective_a4d(self) -> None:
+        x = torch.ones(1, 3, 4, 4)

```

Comment:
Instead of using ones it might be better to use: x = torch.range(1, 3 * 4 * 4).view(1, 3, 4, 4) then you could also assert the values.

Reply:
`torch.range` is being depreciated, so this should be more backwards compatible: 

```
x = torch.arange(1,(3*4*4)+1).view(1, 3, 4, 4).float()
```

Reply:
We also might have to make batch and channel offsets raise a not implemented error for the time being, if I can't figure out how to fix the padding function.

Reply:
Okay, I've added the improved code! It works with everything but a channel or batch size of 2, and I compare the outputs with NumPy in the tests.